### ISSUE TITLE
SEO: Remove rel attribute for own subdomains

### ISSUE THREAD

**Issue by sshcli** [Kunena/Kunena-Forum#6073]
**Steps to reproduce the behavior:**
1. Create a new topic and insert your own subdomain link in the content
For example: `http://sub.example.com`
2. Open the console and take a look to the rel Attribute

**Expected behavior**
No rel attribute if it is your own subdomain.

![examplegood](https://user-images.githubusercontent.com/12942731/43464684-b6accb08-94a9-11e8-9947-c57eb8871ec4.PNG)

**Actual result**
You have rel attribute even in your own subdomains.
`$rel = 'rel="nofollow noopener noreferrer"';`

![examplebad](https://user-images.githubusercontent.com/12942731/43464674-afebb8ce-94a9-11e8-851e-bf3eda2c3edc.PNG)

**System information (please complete the following information)**
Joomla version: 3.8.10
Kunena version: 5.1.3-DEV
Php version: 7.2.8
Database version: MySQL 5.6.39

**Comment by sshcli** [comment]
Please add another "if" for the subdomain:
https://github.com/xillibit/Kunena-forum/blob/4068b34c70d5980fb584c3738004f008213f8ec3/src/components/com_kunena/template/crypsis/layouts/bbcode/url/default.php#L15

**Comment by xillibit** [comment]
So you mean that your browser have the url : www.example.com/forum but in your message you put the following link www.subdomain.example.com/forum ?

**Comment by sshcli** [comment]
> So you mean that your browser have the url : www.example.com/forum but in your message you put the following link www.subdomain.example.com/forum ?

No. What I mean is that the security measure of inserting rel="nofollow noopener noreferrer" to all links in content is not necessary for your own domains or subdomains.  This security measure should only apply for external domains. 
Internal linking is very good for SEO, so, when possible, we should avoid the rel="nofollow" for internal links.
